packages/mobile/src/features/videos/components/Movies/SlaiderLarge.tsx

```diff
@@ -28,28 +38,51 @@ export class SlaiderLarge extends React.Component<any,SlaiderLargeS>{
     })
   }
 
+  public navToSingleMovie = () => {this.props.navigation.navigate(NavigationName.SINGLEMOVIE)}
+
   public Separator = () => (<ViewSeparator/>)
-  public ImgeSlaider = (item:MoviesListSimpleType) => (
-    <View>
-      <ImageSlaider source={{uri: item.uri}}/>
-    </View>
-  )
+  public ImgeSlaider = (item:MoviesListSimpleType, index, x:Animated.Value) => {
```

Comment:
dlugi ten kod moze by to wydzielic jako komponent

Reply:
ok